src/internal/storage/fileset/storage.go

```diff
@@ -479,3 +481,29 @@ func (s *Storage) Pin(tx *pachsql.Tx, fs ID) (PinnedFileset, error) {
 	}
 	return PinnedFileset(*id), nil
 }
+
+type ChunkSetID uint64
+
+func (s *Storage) CreateChunkSet(ctx context.Context, tx *sqlx.Tx) (ChunkSetID, error) {
+	ctx = pctx.Child(ctx, "createChunkset")
+	// Insert ChunkSet into ChunkSet table.
+	var chunksetID ChunkSetID
+	err := tx.GetContext(ctx, &chunksetID, `INSERT INTO storage.chunksets DEFAULT VALUES RETURNING id`)
+	if err != nil {
+		return 0, errors.Wrapf(err, "get chunk set id")
+	}
+
+	// encode chunkset into string for tracker
+	chunksetStrID := "chunkset" + strconv.FormatUint(uint64(chunksetID), 10)
+
+	// List all of the filesets.
+	var pointsTo []string
+	if err = tx.SelectContext(ctx, &pointsTo, `SELECT str_id FROM storage.tracker_objects WHERE str_id LIKE 'fileset%'`); err != nil {
```

Comment:
Create a new `err` binding here, i.e. `:=` instead of `=`.

Reply:
Also, I would include the `/` in your LIKE clause; `fileset/%`.  That is less likely to match something else in the future.